### PR TITLE
Updated index.html

### PR DESCRIPTION
**Pull request by ghost**
Fixed grammatical errors in the header.
Added a space between Orlando and 9002.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <meta name="description" content="18 Dec 2016–28 Oct 2017" />
+  <meta name="description" content="18th Dec 2016 – 28th Oct 2017" />
```
Comment (ByteOfMelon) [inline index.html:8]: Unnecessary change

**Review by ByteOfMelon — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:8]: >Because grammar is unnecessary.

Comment (ghost) [reply index.html:8]: Unnecessary. Closed.
